packages/edit-site/src/components/posts-app/posts-list.js

```diff
@@ -312,20 +316,26 @@ export default function PagePages() {
 		[ totalItems, totalPages ]
 	);
 
-	const { frontPageId, postsPageId, addNewLabel, canCreatePage } = useSelect(
+	const { frontPageId, postsPageId, labels, canCreateRecord } = useSelect(
 		( select ) => {
 			const { getEntityRecord, getPostType, canUser } =
 				select( coreStore );
 			const siteSettings = getEntityRecord( 'root', 'site' );
 			return {
 				frontPageId: siteSettings?.page_on_front,
 				postsPageId: siteSettings?.page_for_posts,
-				addNewLabel: getPostType( 'page' )?.labels?.add_new_item,
-				canCreatePage: canUser( 'create', 'pages' ),
+				labels: getPostType( postType )?.labels,
+				// TODO: check what is the proper way to make this work for any post type..
```

Comment:
To get the resource (posts, pages etc) we can do this:
```
const postTypeObject = getPostType( postType );
const resource = postTypeObject?.rest_base || '';
```

Reply:
Thanks Jorge!